lib/selinuxtest.pm

```diff
@@ -25,6 +25,7 @@ our @EXPORT = qw(
   $file_output
   $policyfile_tar
   download_policy_pkgs
+  check_disabled
```

Comment:
can you try the following in the import list:
```
package selinuxtest;
use base 'Exporter';
use Exporter;
```

And remove `use base "opensusebasetest";`, although I'm not sure about this, maybe you can keep the opensusebasetest base and remove the `use base 'Exporter';`, but at least `use Exporter;` should be there. Please give it a try.

Reply:
I tried multiple combination and this final one is fixing the exporter in `selinuxtest.pm`

Reply:
I tried multiple combination and this final one is fixing the exporter in `selinuxtest.pm`